### ISSUE TITLE
Ultra short command aliases ( e.g. "s" as alias for "sms" in command )

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#222]
```
Like

  - "s" for "sms"
  - "r" for "reply"
  - "c" for "chat"

and so on. You can put your suggestions here.
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 24 Sep 2011 at 2:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 9:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 7 Jan 2012 at 9:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 14 Apr 2012 at 5:41
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
